app/models/qdm/basetypes/data_element.rb

```diff
@@ -68,15 +68,22 @@ def shift_dates(seconds)
         send(field + '=', (send(field).to_time + seconds.seconds).to_datetime) if send(field).is_a? Time
         send(field + '=', send(field).shift_dates(seconds)) if (send(field).is_a? Interval) || (send(field).is_a? DataElement)
 
-        # Special case for facility locations
-        if field == 'facilityLocations'
+        # Special case for facility locations and result
+        if field == 'result'
```

Comment:
@jkotanchik-SB the branch for 'result' is not needed.

The code below takes care of shifting the result.  The branch we were using didn't have that line. 

send(field + '=', (send(field).to_time + seconds.seconds).to_datetime) if send(field).is_a? Time

Reply:
I should have realized that we had already taken care of that with this PR.

https://github.com/projecttacoma/cqm-models/pull/195

Reply:
Then, can I safely remove this whole branch from the the `if` block?

Reply:
Yes you can